src/scene/immediate.js

```diff
@@ -12,50 +12,53 @@ import { MeshInstance } from '../scene/mesh-instance.js';
 
 var identityGraphNode = new GraphNode();
 
-function ImmediateData(device) {
-    this.lineVertexFormat = new VertexFormat(device, [
-        { semantic: SEMANTIC_POSITION, components: 3, type: TYPE_FLOAT32 },
-        { semantic: SEMANTIC_COLOR, components: 4, type: TYPE_UINT8, normalize: true }
-    ]);
-    this.lineBatches = [];
-    this.layers = [];
-    this.layerToBatch = {};
-    this.quadMesh = null;
-    this.cubeLocalPos = null;
-    this.cubeWorldPos = null;
-    this.identityGraphNode = new GraphNode();
-}
+class ImmediateData {
+
```

Comment:
Just as a reminder, I didn't insert this newline in the classes I've migrated. We should be consistent. I just followed the convention used on MDN:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes